master/static/migrations/20231129123059_run-trial-v2-create.tx.up.sql

```diff
@@ -0,0 +1,67 @@
+-- Is this run_id a bad idea as a primary key?
+-- Would it make sense for "possible performance"
+-- reasons to use a an id that won't have gaps in the sequence.
+-- My only issue is then a field called id could be mistaked for trial id.
+CREATE TABLE trials_v2 (
+    run_id integer PRIMARY KEY REFERENCES runs(id) ON DELETE CASCADE,
+    request_id text,
+    runner_state text NOT NULL DEFAULT 'UNSPECIFIED',
+    seed integer NOT NULL DEFAULT 0
+);
+
+-- TODO how expensive will this be? (Likely not that expesive???)
+INSERT INTO trials_v2 (run_id, request_id, runner_state, seed)
```

Comment:
Do you think this is a potential problem?

```
EXPLAIN ANALYZE INSERT INTO trials_v2 (run_id, request_id, runner_state, seed)
determined-> SELECT id, request_id, runner_state, seed
determined-> FROM runs;
                                                   QUERY PLAN                                                    
-----------------------------------------------------------------------------------------------------------------
 Insert on trials_v2  (cost=0.00..6346.98 rows=32298 width=56) (actual time=333.039..333.039 rows=0 loops=1)
   ->  Seq Scan on runs  (cost=0.00..6346.98 rows=32298 width=56) (actual time=0.011..28.982 rows=32307 loops=1)
 Planning Time: 0.570 ms
 Trigger for constraint trials_v2_run_id_fkey: time=1263.591 calls=32307
 Execution Time: 1599.575 ms
(5 rows)
```

1599.575 ms / 32298 trials => 0.04952551241ms per trial, assuming worse case is 500 seconds or a little over 8 minutes for 1 million trials